packages/clerk-js/src/ui/components/OrganizationProfile/OrganizationSettings.tsx

```diff
@@ -58,9 +58,13 @@ const OrganizationProfileSection = () => {
 };
 
 const OrganizationDangerSection = () => {
-  const { organization, membership } = useCoreOrganization();
+  const { organization, membership, membershipList } = useCoreOrganization({
+    membershipList: {},
+  });
   const { navigate } = useRouter();
 
+  const hasMoreThanOneAdmin = (membershipList || [])?.filter(m => m.role === 'admin')?.length > 1;
```

Comment:
🚫 I believe this will not work. `membershipList` will not  contain all members 

The following code, will fetch memberships with a default limit and a default offset.
```ts
const { organization, membership, membershipList } = useCoreOrganization({
    membershipList: {},
  });  
```

For BAPI the default values are mentioned [here](https://clerk.com/docs/reference/backend-api/tag/Organization-Memberships#operation/ListOrganizationMemberships). I couldn't find the default values for FAPI

Reply:
@panteliselef you are right, we probably need to have the limit higher like 50 or something. Ideally, we would want to filter by role and get only the admins but I know we don't have that functionality yet. cc @chanioxaris 

I think we are ok limiting it to 50 since I have the suspicion we don't have an active organization with more members.

Reply:
Yes Pantelis is right. I would suggest to ignore this for now, until we introduce support for filtering by role to make it more future proof. I am gonna create a ticket to tackle this asap

Reply:
@panteliselef @chanioxaris reverting this change since it will be handle in another ticket by team organization